### DISCUSSION TITLE
Pushing  DJI to fix their OSD

### DISCUSSION THREAD

**Discussion by donne1226** [iNavFlight/inav#10154] [Ideas]
Hi, Darren @MrD-RC 
I read the email you sent to DJI. Thank you very much for your efforts.
https://www.mrd-rc.com/blog/trying-to-help-dji-fix-their-osd-for-everyone/

But I think the power of one person is too small, I saw several people including me on the official forum of DJI Chinese post about optimizing OSD, but did not get a response from the official staff. Even when I posted a screenshot of the WTFOS OSD, it was immediately deleted. They should have known about this a long time ago. They don't care how we feel. Even you're an INAV developer(you didn't mentioned that in the Email though), they won't take it seriously! I feel like more people need to get involved and make more of us heard at DJI. Otherwise, they still won't make any changes.

My idea was to make a video and make it widely disseminated, and everybody reposted it to video platforms like YouTube, Ticktok, Bilibili (where most of the Chinese pilots post videos). 

The video reads presumably:
1. Compare WTFOS OSD or even Betaflight. (some people don't know that DJI can do better)
2. Make it clear that only DJI can solve this problem. The responsibility lies with DJI
3. If they don't make changes, players will collectively boycott O4 air unit and Goggles 3. (I think that's the most important point, they will only make changes if it affects them to make money)

**Comment by jeffythequick** [comment]
Yeah, Walksnail is looking better and better, but I want to see INav Goggles videos before committing.  WTFOS/V1 worked great for my Goggles V2, and now that I'm on Goggles 2/3 and the DJI V2, if I could, I'd send them back.

DJI should do one of these things:
1.  Fix the OSD interface/API for programmers
2. Open the code up so programmers can code the DJI OSD
3. Give us the link to Walksnail so we don't have to look it up ourselves.

**Reply by MrD-RC** [reply]
I doubt any of those 3 will happen. 

https://www.mrd-rc.com/blog/trying-to-help-dji-fix-their-osd-for-everyone/

But here's the Walksnail link for convenience 🤣 https://caddxfpv.com/collections/walksnail-avatar-system
